pkg/sloop/queries/eventquery.go

```diff
@@ -36,8 +37,24 @@ func GetEventData(params url.Values, t typed.Tables, startTime time.Time, endTim
 	err := t.Db().View(func(txn badgerwrap.Txn) error {
 		var err2 error
 		var stats typed.RangeReadStats
+		selectedNamespace := params.Get(NamespaceParam)
+		selectedName := params.Get(NameParam)
+		selectedKind := params.Get(KindParam)
+		if selectedKind == kubeextractor.NodeKind || selectedKind == kubeextractor.NamespaceKind {
```

Comment:
I suspect we have the logic elsewhere, and we are going to add to it.  Node and Namespace are both cluster-scoped resources, but there are also PSP, ClusterRole, ClusterRoleBinding, Webhooks, etc.  Can you make a function in kubeExtractor like isClustersScopedResource() and use that here and other places with the same logic?

Reply:
yeah, we can make a function to do this.